### DISCUSSION TITLE
[Feat Req]- Duplicate/Clone Overrides (in Time Series)

### DISCUSSION THREAD

**Discussion by bob454522** [grafana/grafana#41384] [Dashboards, Panels & Visualizations]
(nb: im mainly referencing graphite as datasource in below, but also influxDB)
grafana version = 8.2.2

[Feature Request]: a Duplicate/Clone (button) for "Overrides" entries:
There are many times we have to create Override entries using regex, which are only different in 1 or 2 elements (ie the regex is slightly different, and/or overrides to a different color or threshold values)

Having the ability to duplicate or clone an existing Override entry would be very helpful.

Currently to make a 2nd Override (very similar to the 1st override), it will take several actions / clicks, this could be reduced greatly via a clone option.
here is where im referring to (ie have a duplicate/clone button next to the trash icon):

![image](https://user-images.githubusercontent.com/60830628/140582383-774489e8-4383-48a3-ad50-734f674757a2.png)

(im happy to see more elements moved over to the Override function, ie as of recent grafana releases, most of the "Panel -> Series Override" elements have been condensed/move to the "Override" tab - this is GREAT! )

Note- my original feature request for this, a few months ago, was closed and referred to a duplicate https://github.com/grafana/grafana/issues/27305
(which then also was closed "as it's for the old graph panel that we are no longer extending / adding features to" , however this request really applies to any/all "Overrides" in grafana-  but mainly to the new, awesome, "Time Series" panel at this point in time- so im re-opening it)

Ability to clone an override (and then make changes to the clone) , would really be helpful as it helps keep consistency and a "consistent look" to a graph or panel.  Cloning Overrides can also help with the grafana UI / organization as it will keep individual override elements in the same order / flow.  

(Example: i frequently will use the same override elements in 2 or more overrides,  but they will appear out of order in the UI, as the order displayed, is based upon the order in which you add Override sub-elements). (understandably)  
Ie overrideA makes use of:
 Soft-Min, Soft-Max, Unit, Color Scheme  (in that order, in the unit)

a few days later i add in:
overrideB  but i add sub-elements in this exact order:
 Color Scheme , Unit , Soft-Max, Soft-Min (in that order, in the unit).

At a later date, If i want to change Both override A and B's "Soft-Max" , they appear in a different order in the UI. (understandably).  However a clone Override feature would help improve this ordering mix up.

Thank you grafana team!
LOVE GRAFANA!

**Comment by SirFlour** [comment]
That feature would be really helpful :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
